src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2607,4 +2607,10 @@
   <data name="MultiSelect" xml:space="preserve">
     <value>Multiselect</value>
   </data>
+  <data name="Hashes" xml:space="preserve">
+    <value>Hashes</value>
+  </data>
+  <data name="HashesUnavailable" xml:space="preserve">
+    <value>Hashes are unavailable</value>
```

Comment:
When will this show?

Reply:
When file cannot be accessed == When File.Open() throw an exception

Reply:
Shouldn't the tab be hidden?

Reply:
That's good idea.